client/src/Pages/Home.js

```diff
@@ -3,7 +3,8 @@ import logo from '../logo.svg';
 import '../App.css';
 import styled from 'styled-components'
 import { useMediaQuery } from 'react-responsive'
-
+import {SearchButton, SearchButtonDiv} from './HomeStyles.js'
+import { ColorButton } from "../components/Create.styles";
 const Home = () => {
     const MainHeader = styled.header`
         display: flex;
```

Comment:
I would move this styled component into the styles file to keep things consistent.

Reply:
Done